### ISSUE TITLE
Broken Build due to com.github.spotbugs.plugin.eclipse-*.jar!/META-INF/MANIFEST.MF: Invalid manifest header Bundle-ClassPath: "spotbugs-plugin.jar,"

### ISSUE THREAD

**Issue by vorburger** [m2e-code-quality/m2e-code-quality#227]
#226 re. _Bump tycho-version from 1.5.1 to 1.6.0_ from DependaBot [failed on TravisCI](https://travis-ci.org/m2e-code-quality/m2e-code-quality/builds/639785899?utm_source=github_status&utm_medium=notification) due to:

```
[INFO] Resolving class path of MavenProject: com.basistech.m2e-code-quality:com.basistech.m2e.code.quality.spotbugs:1.1.5-SNAPSHOT @ /home/travis/build/m2e-code-quality/m2e-code-quality/com.basistech.m2e.code.quality.spotbugs/pom.xml
[ERROR] Internal error: org.eclipse.tycho.core.osgitools.OsgiManifestParserException: Exception parsing OSGi MANIFEST /home/travis/.m2/repository/p2/osgi/bundle/com.github.spotbugs.plugin.eclipse/4.0.0.202001200351-6732852/com.github.spotbugs.plugin.eclipse-4.0.0.202001200351-6732852.jar!/META-INF/MANIFEST.MF: Invalid manifest header Bundle-ClassPath: "spotbugs-plugin.jar," -> [Help 1]
org.apache.maven.InternalErrorException: Internal error: org.eclipse.tycho.core.osgitools.OsgiManifestParserException: Exception parsing OSGi MANIFEST /home/travis/.m2/repository/p2/osgi/bundle/com.github.spotbugs.plugin.eclipse/4.0.0.202001200351-6732852/com.github.spotbugs.plugin.eclipse-4.0.0.202001200351-6732852.jar!/META-INF/MANIFEST.MF: Invalid manifest header Bundle-ClassPath: "spotbugs-plugin.jar,"
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:120)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: org.eclipse.tycho.core.osgitools.OsgiManifestParserException: Exception parsing OSGi MANIFEST /home/travis/.m2/repository/p2/osgi/bundle/com.github.spotbugs.plugin.eclipse/4.0.0.202001200351-6732852/com.github.spotbugs.plugin.eclipse-4.0.0.202001200351-6732852.jar!/META-INF/MANIFEST.MF: Invalid manifest header Bundle-ClassPath: "spotbugs-plugin.jar,"
    at org.eclipse.tycho.core.osgitools.OsgiManifest.getManifestElements (OsgiManifest.java:171)
    at org.eclipse.tycho.core.osgitools.OsgiManifest.parseBundleClasspath (OsgiManifest.java:179)
    at org.eclipse.tycho.core.osgitools.OsgiManifest.<init> (OsgiManifest.java:53)
    at org.eclipse.tycho.core.osgitools.OsgiManifest.parse (OsgiManifest.java:152)
    at org.eclipse.tycho.core.osgitools.DefaultBundleReader.loadManifestFromFile (DefaultBundleReader.java:83)
    at org.eclipse.tycho.core.osgitools.DefaultBundleReader.doLoadManifest (DefaultBundleReader.java:62)
    at org.eclipse.tycho.core.osgitools.DefaultBundleReader.loadManifest (DefaultBundleReader.java:51)
    at org.eclipse.tycho.core.osgitools.EquinoxResolver.loadManifest (EquinoxResolver.java:269)
    at org.eclipse.tycho.core.osgitools.EquinoxResolver.newState (EquinoxResolver.java:161)
    at org.eclipse.tycho.core.osgitools.EquinoxResolver.newResolvedState (EquinoxResolver.java:69)
    at org.eclipse.tycho.core.osgitools.OsgiBundleProject.getResolverState (OsgiBundleProject.java:263)
    at org.eclipse.tycho.core.osgitools.OsgiBundleProject.resolveClassPath (OsgiBundleProject.java:164)
    at org.eclipse.tycho.core.resolver.DefaultTychoResolver.resolveProject (DefaultTychoResolver.java:148)
    at org.eclipse.tycho.core.maven.TychoMavenLifecycleParticipant.afterProjectsRead (TychoMavenLifecycleParticipant.java:95)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:264)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: org.osgi.framework.BundleException: Invalid manifest header Bundle-ClassPath: "spotbugs-plugin.jar,"
    at org.eclipse.osgi.util.ManifestElement.parseHeader (ManifestElement.java:355)
    at org.eclipse.tycho.core.osgitools.OsgiManifest.getManifestElements (OsgiManifest.java:169)
    at org.eclipse.tycho.core.osgitools.OsgiManifest.parseBundleClasspath (OsgiManifest.java:179)
    at org.eclipse.tycho.core.osgitools.OsgiManifest.<init> (OsgiManifest.java:53)
    at org.eclipse.tycho.core.osgitools.OsgiManifest.parse (OsgiManifest.java:152)
    at org.eclipse.tycho.core.osgitools.DefaultBundleReader.loadManifestFromFile (DefaultBundleReader.java:83)
    at org.eclipse.tycho.core.osgitools.DefaultBundleReader.doLoadManifest (DefaultBundleReader.java:62)
    at org.eclipse.tycho.core.osgitools.DefaultBundleReader.loadManifest (DefaultBundleReader.java:51)
    at org.eclipse.tycho.core.osgitools.EquinoxResolver.loadManifest (EquinoxResolver.java:269)
    at org.eclipse.tycho.core.osgitools.EquinoxResolver.newState (EquinoxResolver.java:161)
    at org.eclipse.tycho.core.osgitools.EquinoxResolver.newResolvedState (EquinoxResolver.java:69)
    at org.eclipse.tycho.core.osgitools.OsgiBundleProject.getResolverState (OsgiBundleProject.java:263)
    at org.eclipse.tycho.core.osgitools.OsgiBundleProject.resolveClassPath (OsgiBundleProject.java:164)
    at org.eclipse.tycho.core.resolver.DefaultTychoResolver.resolveProject (DefaultTychoResolver.java:148)
    at org.eclipse.tycho.core.maven.TychoMavenLifecycleParticipant.afterProjectsRead (TychoMavenLifecycleParticipant.java:95)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:264)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
```

**Comment by vorburger** [comment]
Looks like this is probably related to https://github.com/spotbugs/spotbugs/issues/1067 (note the Eclipse plugin with the bad MANIFEST is NOT the ours from m2e-code-quality but the SpotBugs one).

Not sure if the Tycho upgrade is causing it (more strict), or just make it appear, because we don't rebuild this very often, but I suspect it's the latter and even a current build without changes would may fail like this (haven't checked though).

**Comment by vorburger** [comment]
While https://github.com/spotbugs/spotbugs/issues/1067 has been closed today, this issue is still open for this project, because https://spotbugs.github.io/eclipse/ does not, yet, host a build including that fix (but still 4.0.0.202001200351-6732852) - so PR #226 [still failed to build](https://travis-ci.org/m2e-code-quality/m2e-code-quality/builds/639785899) (with above).

**Comment by vorburger** [comment]
https://spotbugs.github.io/eclipse/ now has a com.github.spotbugs.plugin.eclipse Version 4.0.0.202002080054-d820dad, and the build is no longer blocked by the problem above. There's now a completely unrelated new problem, see #229.
